docs_src/manuals/getting_started/code/CMakeLists.txt.in

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION @CMAKE_MINIMUM_REQUIRED_VERSION@)
+cmake_minimum_required(VERSION 3.14.5)
```

Comment:
For some reason ```@CMAKE_MINIMUM_REQUIRED_VERSION@``` turned into ```3.3``` which had effect on how MPI being found in ```find_dependency( MPI )```  ( within ```find_package( GridTools )``` ).
```3.14.5``` is the version specified in the top CMakeLists.txt

Reply:
I reverted this. I propose we fix this issue in a separate PR, if you cannot find a workaround on your side.